diff --git a/Python/asap3/Internal/ParallelListOfAtoms.py b/Python/asap3/Internal/ParallelListOfAtoms.py
index 2adf01d8..6efcf54d 100644
--- a/Python/asap3/Internal/ParallelListOfAtoms.py
+++ b/Python/asap3/Internal/ParallelListOfAtoms.py
@@ -63,7 +63,7 @@ class ParallelAtoms(ase.Atoms):
         self._calc = None
         if Cell is not None:
             # Needed in newest ASE
-            self._cellobj = Cell.new(pbc=False)
+            self._cellobj = Cell.new()
 
         # Extract all data from the atoms object.  Not nice :-)
         self.arrays = {}
