diff --git a/setup.py b/setup.py
index 03348c6d..8c55c133 100644
--- a/setup.py
+++ b/setup.py
@@ -293,7 +293,7 @@ def build_interpreter(compiler, common_src, parallel_src,
     lib_dirs = ' '.join(['-L' + lib for lib in library_dirs])
 
     libs = ' '.join(['-l' + lib for lib in libraries if lib.strip()])
-    libs += ' ' + cfgDict.get('BLDLIBRARY', '-lpython%s' % cfgDict['VERSION'])
+    libs += ' ' + cfgDict.get('BLDLIBRARY', '-lpython%s' % cfgDict['VERSION']).replace('libpython','-lpython').replace('.a','')
     libs += ' ' + ' '.join([libs, cfgDict['LIBS'], cfgDict['LIBM']])
     if sys.platform == 'darwin':
         runtime_libs = ''   # No -rpath on macOS
